docs/INSTALL.md

```diff
@@ -0,0 +1,674 @@
+
+# Are users allowed to set their own From: address?
+# YES - Allow the user to specify their own From: address
+# NO - Use the system generated From: address
+FromLineOverride=YES
+```
+
+<a name="install-other-depedencies"></a>
+## Install other Depedencies
+
+On Debian/Ubuntu execute: 
+```
+apt-get install zip unzip php5-gd php5-mysql php-xsl imagemagick openjdk-7-jdk
+```
+
+On RHEL/CentOS execute: 
+```
+yum install zip unzip php-gd libxslt ImageMagick java-1.7.0-openjdk
+```
+<a name="installation-wizard"></a>
+## Installation Wizard
+
+After having all dependencies installed and configured you should be able to run the ILIAS Installation Wizard using http://yourservername.org/setup/setup.php
+
+Make sure to reload your Apache configuration before entering the Wizard. Otherwise there are unmet dependencies in the setup (like XLS and GD are both installed but ILIAS does not see them, yet).
+
+<a name="configure-ilias-java-rpc-server-optional"></a>
+## Configure ILIAS Java RPC server (OPTIONAL)
+
+The ILIAS Java RPC server is used for certain OPTIONAL functions as Lucene Search or generating PDF Certificates. To enable the RPC server you need to place a configuration file in ```<YOUR_ILIAS_DIR>/Services/WebServices/RPC/lib/ilServer.properties```:
+
+```
+[Server]
+IpAddress = localhost
+Port = 11111
+IndexPath = /var/www/html/ilias/data/
+LogFile = /var/www/files/ilServer.log
+LogLevel = WARN
+NumThreads = 1
+RamBufferSize = 256
+IndexMaxFileSizeMB = 500
+
+[Client1]
+ClientId = ACMECorp
+NicId = 0
+IliasIniPath = /var/www/html/ilias/ilias.ini.php
+```
+
+ILIAS can generate a proper configuration file via the Administration menu ("Administration -> General Settings -> Server -> Java-Server -> Create Configuration File"). Please note that the configuration file is not directly written to the file system, you MUST copy the displayed content and create the file manually.
+
+You MAY use the following SysV-Initscript to start the RPC server:
+
+```
+#!/bin/bash
+
+JAVABIN=/usr/bin/java    # Type in the path to your java binary
+ILIASDIR=/var/www/html/ilias    # Type in the root directory of your ILIAS installation
+IL_SERVER_INI=/var/www/html/ilias/Services/WebServices/RPC/lib/ilServer.properties    # Type in the location of the RPC config file
+
+case "$1" in
+    start)
+        echo "Starting ILIAS Java-Server"
+        $JAVABIN -Dfile.encoding=UTF-8 -jar $ILIASDIR/Services/WebServices/RPC/lib/ilServer.jar $IL_SERVER_INI start &
+        exit 0
+        ;;
+
+    stop)
+        echo "Shutting down ILIAS Java-Server"
+        $JAVABIN -jar $ILIASDIR/Services/WebServices/RPC/lib/ilServer.jar $IL_SERVER_INI stop
+        exit 0
+        ;;
+
+    status)
+        $JAVABIN -jar $ILIASDIR/Services/WebServices/RPC/lib/ilServer.jar $IL_SERVER_INI status
+        ;;      
+
+    restart)
+        $0 stop
+        sleep 2
+        $0 start
+        ;;
+
+    *)
+        echo "Usage: $0 {start|stop|status|restart}"
+        exit 1
+esac
+
+exit 0
+```
```

Comment:
Works perfectly well, but since the Server doesn't do anything yet, maybe a link to the README.md (<YOUR_ILIAS_DIR>/Services/WebServices/RPC/lib/README.txt) would come handy for people who want to install lucene right away.

Something like: If you want to install lucene, follow the instructions in the provided readme...

Otherwise then that, it's a great installation document and should help quite a lot ;)